### PR TITLE
Add xcao-mms-Waltham.md file

### LINKED ISSUE #13: Create a new file for xcao-mms - Waltham
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a PR
- [ ] Discuss
- [ ]  Merge PR

### PR DESCRIPTION
**Pull request by xcao-mms**
closes #13

### CODE REVIEW

File: xcao-mms-Waltham.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Waltham, MA
+
```
Comment (mbayless) [inline xcao-mms-Waltham.md:2]: I work with you!!!!! do you remember me

File: xcao-mms-Waltham.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+### Recommendations
+- Eat at Main Street!
```
Comment (xcao-mms) [inline xcao-mms-Waltham.md]: Which restaurant?

File: xcao-mms-Waltham.md (hunk 1 above)
Comment (xcao-mms) [reply xcao-mms-Waltham.md:2]: no, and who are you??

Comment (xcao-mms) [reply xcao-mms-Waltham.md:2]: :smile_cat:

**Review by xcao-mms — COMMENTED** [review]
good to go

**Pull request merged**